code/core/src/core-server/utils/save-story/save-story.ts

```diff
@@ -49,6 +49,8 @@ const removeExtraNewlines = (code: string, name: string) => {
     : code;
 };
 
+const IS_CLI_EXAMPLE = /^example-(button|header|page)--/;
```

Comment:
Do we have a more formal version of this somewhere?

Reply:
Good call